crates/transaction/manifests/src/manifests_security_shield/access_controller_factors_and_time_input.rs

```diff
@@ -0,0 +1,100 @@
+use crate::prelude::*;
+use radix_engine_interface::blueprints::access_controller::{
+    AccessControllerInitiateRecoveryAsPrimaryInput as ScryptoAccessControllerInitiateRecoveryAsPrimaryInput,
+    AccessControllerInitiateRecoveryAsRecoveryInput as ScryptoAccessControllerInitiateRecoveryAsRecoveryInput,
+    AccessControllerQuickConfirmPrimaryRoleRecoveryProposalInput as ScryptoAccessControllerQuickConfirmPrimaryRoleRecoveryProposalInput,
+    AccessControllerQuickConfirmRecoveryRoleRecoveryProposalInput as ScryptoAccessControllerQuickConfirmRecoveryRoleRecoveryProposalInput,
+    AccessControllerTimedConfirmRecoveryInput as ScryptoAccessControllerTimedConfirmRecoveryInput,
+};
+
+#[derive(Debug, Clone)]
+pub struct AccessControllerFactorsAndTimeInput {
```

Comment:
shared type to create:
- AccessControllerInitiateRecoveryAsPrimaryInput
- AccessControllerInitiateRecoveryAsRecoveryInput
- AccessControllerQuickConfirmPrimaryRoleRecoveryProposalInput
- AccessControllerQuickConfirmRecoveryRoleRecoveryProposalInput
- AccessControllerTimedConfirmRecoveryInput

Input to `call_method` instruction, using `SecurityStructureOfFactorInstances` - which is `Into<RuleSet>` (and time u32).